Bio/SeqIO/PdbIO.py

```diff
@@ -34,22 +52,13 @@ def AtomIterator(pdb_id, structure):
     See Bio.SeqIO.PdbIO.PdbAtomIterator and Bio.SeqIO.PdbIO.CifAtomIterator for
     details.
     """
-    from Bio.SeqUtils import seq1
-
-    def restype(residue):
-        """Return a residue's type as a one-letter code.
-
-        Non-standard residues (e.g. CSD, ANP) are returned as 'X'.
-        """
-        return seq1(residue.resname, custom_map=protein_letters_3to1)
-
     model = structure[0]
     for chn_id, chain in sorted(model.child_dict.items()):
         # HETATM mod. res. policy: remove mod if in sequence, else discard
         residues = [
             res
             for res in chain.get_unpacked_list()
-            if seq1(res.get_resname().upper(), custom_map=protein_letters_3to1) != "X"
+            if _res2aacode(res.get_resname().upper(), undef_code="X") != "X"
```

Comment:
Having the "X" explicit here and in the private function definition seems redundant and potentially confusing. Here you set the undefined code, but 20 lines later you do not.

I would suggest making the function take two required arguments, and then all the calls to it will be consistent. Or since you always use "X" just hard code it, no second argument?